toolkit/src/main/cpp/export/meta_toolkit_export_plugin.cpp

```diff
@@ -88,15 +110,33 @@ Dictionary MetaToolkitExportPlugin::_get_export_options_overrides(
 	}
 
 	// Check if this plugin is enabled
-	if (!_get_bool_option("meta_toolkit/enable_meta_toolkit")) {
+	if (!_is_plugin_enabled()) {
 		return overrides;
 	}
 
-	// Gradle build overrides
-	overrides["gradle_build/use_gradle_build"] = true;
-	overrides["gradle_build/export_format"] = 0; // apk
-	overrides["gradle_build/min_sdk"] = "29"; // Android 10
-	overrides["gradle_build/target_sdk"] = "32"; // Android 12
+	// Disable gradle build on the Android editor
+	bool is_mobile_editor = OS::get_singleton()->has_feature("mobile");
+	if (!is_mobile_editor) {
+		// Gradle build overrides
+		overrides["gradle_build/use_gradle_build"] = true;
+		overrides["gradle_build/export_format"] = 0; // apk
+		overrides["gradle_build/min_sdk"] = "29"; // Android 10
+		overrides["gradle_build/target_sdk"] = "32"; // Android 12
+	} else {
+		overrides["gradle_build/use_gradle_build"] = false;
```

Comment:
Let's remove this given the progress being made on the gradle build and let's have the `min_sdk` and `target_sdk` reflect the values from above.